mne/beamformer/_compute_beamformer.py

```diff
@@ -123,10 +123,13 @@ def _check_one_ch_type(info, picks, noise_cov):
     info['comps'] = comps
     ch_types =\
         [_contains_ch_type(info_pick, tt) for tt in ('mag', 'grad', 'eeg')]
-    if sum(ch_types) > 1 and noise_cov is None:
+    if beamformer == 'lcmv' and sum(ch_types) > 1 and noise_cov is None:
```

Comment:
beamformer -> method

to be consistent with MNE solvers?

Reply:
+1 
will change